packages/react-core/src/core/providers/thirdweb-provider.tsx

```diff
@@ -117,33 +114,22 @@ export const ThirdwebProviderCore = <
 >(
   props: React.PropsWithChildren<ThirdwebProviderCoreProps<TChains>>,
 ) => {
-  // deprecations
-  if (props.chainRpc) {
-    showDeprecationWarning("chainRpc", "supportedChains");
-  }
-
   const supportedChains =
     props.supportedChains || (defaultChains as any as TChains);
 
   const dAppMeta = props.dAppMeta || defaultdAppMeta;
+
   const activeChainObj = useMemo(() => {
-    if (!props.activeChain) {
-      return supportedChains[0];
-    }
     if (typeof props.activeChain === "number") {
-      return (
-        supportedChains.find((chain) => chain.chainId === props.activeChain) ||
-        supportedChains[0]
+      return supportedChains.find(
+        (chain) => chain.chainId === props.activeChain,
       );
     }
     if (typeof props.activeChain === "string") {
-      return (
-        supportedChains.find((chain) => chain.slug === props.activeChain) ||
-        supportedChains[0]
-      );
+      return supportedChains.find((chain) => chain.slug === props.activeChain);
     }
 
-    return props.activeChain || supportedChains[0] || defaultChains[0];
+    return props.activeChain;
```

Comment:
With this change we no longer have an `activeChain` in our SDK by default. Not sure about the downstream consequences here. cc: @jnsdls

Reply:
`WrappedThirdwebSDKProvider` handles the case where activeChain is `undefined` 

<img width="890" alt="image" src="https://user-images.githubusercontent.com/22043396/225316312-5f03b527-4d0d-4ccf-8b49-9614f1f03af1.png">